hack/jenkins/e2e.sh

```diff
@@ -192,6 +192,19 @@ case ${JOB_NAME} in
     MINION_SIZE="n1-standard-1"
     NUM_MINIONS="2"
     ;;
+
+  # Runs non-flaky tests on GCE on the release-latest branch,
+  # sequentially. As a reminder, if you need to change the skip list
```

Comment:
Yeah, it would checkout a git workspace at release-0.21, run `hack/jenkins/build.sh` and `hack/jenkins/e2e.sh`. (Until we decide we need to fan out the release build jobs as well).